staking/availability/measure.go

```diff
@@ -215,3 +221,37 @@ func ComputeAndMutateEPOSStatus(
 
 	return nil
 }
+
+// UpdateMinimumCommissionFee update the validator commission fee to the minimum 5%
+// if the validator has a lower commission rate and 100 epochs have passed after
+// the validator was first elected.
+func UpdateMinimumCommissionFee(
+	electionEpoch *big.Int,
+	state *state.DB,
+	addr common.Address,
+) error {
+	utils.Logger().Info().Msg("begin update min commission fee")
+
+	wrapper, err := state.ValidatorWrapper(addr)
+	if err != nil {
+		return err
+	}
+
+	firstElectionEpoch := state.GetValidatorFirstElectionEpoch(addr)
+
+	promoPeriod := 100
+	if nodeconfig.GetDefaultConfig().GetNetworkType() != nodeconfig.Mainnet {
+		promoPeriod = 10
```

Comment:
promoPeriod = 10 is random value right? Because except for the main network which is the first 100 epochs(proposal descrition), other networks don't care much about this value

Reply:
yes, just want to make it faster so we can test it early.